src/NodaTime.Demo/PeriodBuilderDemo.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2010 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NUnit.Framework;
+
+namespace NodaTime.Demo
+{
+    public class PeriodBuilderDemo
+    {
+        [Test]
+        public void ConstructionFromPeriod()
+        {
+            Period existingPeriod = Period.FromYears(5);
+            PeriodBuilder periodBuilder = Snippet.For(new PeriodBuilder(existingPeriod));
+            Period period = periodBuilder.Build();
```

Comment:
It would probably be worth mutating this, e.g. setting `periodBuilder.Months`, then keep the existing assertion about `period.Years`, but add another one for `period.Months` as well.